core/lib/dal/src/blocks_dal.rs

```diff
@@ -2394,6 +2396,29 @@ impl BlocksDal<'_, '_> {
         .flatten())
     }
 
+    pub async fn get_sealed_at(
+        &mut self,
+        l1_batch_number: L1BatchNumber,
+    ) -> DalResult<Option<NaiveDateTime>> {
```

Comment:
Bikeshedding: Most existing DAL methods convert time to `DateTime<Utc>`.

Reply:
fixed